save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -114,6 +129,15 @@ class OrganizationService(
                 project.status == CREATED
             }
 
+    /**
+     * @param organizationName
+     * @param status
+     * @param authentication
+     * @return whether user described by [authentication] can have permission on change [organization] status on [newStatus] depending on the number of projects in the organization
+     */
+    fun canChangeStatus(organizationName: String, status: OrganizationStatus, authentication: Authentication) =
```

Comment:
we have organizationPermissionEvaluator to check user's permission. such logic should be hidden there

Reply:
Deleted this logic